client/src/templates/Challenges/components/help-modal.tsx

```diff
@@ -53,61 +89,208 @@ function HelpModal({
+                  {t('learn.must-confirm-statements')}
+                </legend>
+                <CheckboxHelpModal
+                  name='read-search-ask-checkbox'
+                  i18nkey={t('learn.read-search-ask-checkbox')}
+                  onChange={event =>
+                    handleCheckboxChange(event, setReadSearchCheckbox)
+                  }
+                  value={readSearchCheckbox}
+                  href={RSA}
+                  t={t}
+                />
+                <Spacer size='small' />
+                <CheckboxHelpModal
+                  name='similar-questions-checkbox'
+                  i18nkey={t('learn.similar-questions-checkbox')}
+                  onChange={event =>
+                    handleCheckboxChange(event, setSimilarQuestionsCheckbox)
+                  }
+                  value={similarQuestionsCheckbox}
+                  href={generateSearchLink(challengeTitle, challengeBlock)}
+                  t={t}
+                />
+              </fieldset>
+              <label htmlFor='help-modal-form-description'>
+                {t('forum-help.whats-happening')}
+                <span className='sr-only'>50 character minimum</span>
+              </label>
+              <FormControl
+                id='help-modal-form-description'
+                name='description'
+                onChange={(event: React.ChangeEvent<HTMLInputElement>) => {
+                  setDescription(event.target.value);
+                }}
+                componentClass='textarea'
+                required={true}
+                style={{
+                  height: 100
+                }}
+                value={description}
+                maxLength={DESCRIPTION_MAX}
+                type='text'
+              />
+              {description.length < DESCRIPTION_MIN ? (
+                <p className='chars-left'>
+                  {t('learn.minimum-characters').replace(
+                    '{{characters}}',
+                    (DESCRIPTION_MIN - description.length).toString()
+                  )}
+                </p>
+              ) : (
+                <p className='chars-left'>
+                  {t('learn.characters-left').replace(
+                    '{{characters}}',
+                    (DESCRIPTION_MAX - description.length).toString()
+                  )}
+                </p>
+              )}
+            </div>
+            <Spacer size='xxSmall' />
+
+            <Button
+              block={true}
+              size='large'
+              variant='primary'
+              type='submit'
+              disabled={!canSubmitForm()}
+            >
+              {t('buttons.submit')}
+            </Button>
+            <Spacer size='xxSmall' />
+            <Button
+              block={true}
+              size='large'
+              variant='primary'
+              onClick={() => {
+                setShowHelpForm(false);
+              }}
+            >
+              {t('buttons.back')}
```

Comment:
```suggestion
              onClick={closeHelpModal}
            >
              {t('buttons.cancel')}
```

Cancel button on second dialog instead of Back button.